recipes/onedpl/all/conanfile.py

```diff
@@ -28,6 +32,9 @@ def _source_subfolder(self):
     def configure(self):
         if self.settings.compiler.cppstd:
             tools.check_min_cppstd(self, 11)
+        if self.options.backend == "tbb":
+            # Listed as an OPTIONAL_COMPONENT but optional components not supported by conan
+            self.options["tbb"].tbbmalloc = True
```

Comment:
Please also raise an exception in validate if `self.options.backend == "tbb"` and `not self.options["tbb"].tbbmalloc`, because the option values set in `configure` can be changed by other recipes